pinot-core/src/main/java/com/linkedin/pinot/core/realtime/converter/stats/RealtimeColumnStatistics.java

```diff
@@ -67,21 +68,64 @@ public RealtimeColumnStatistics(RealtimeColumnDataSource dataSource, int[] sorte
 
   @Override
   public Object getMinValue() {
+    if (_dictionaryReader == null) {
```

Comment:
I understand this is temporary. But just for my understanding, is there any code that actually needs these values in the 'no-dictionary' flow?